packages/ember-toucan-core/src/components/form/file-input/list-item.hbs

```diff
@@ -8,18 +8,17 @@
     }}"
   ...attributes
 >
-  <div class="flex-1 px-4 py-1">
-    <p
-      class="text-titles-and-attributes type-md-tight m-0 truncate p-0"
-      data-file-name
-    >{{@file.name}}</p>
+  {{! The `pl-2` added here is to offset the spacing added by the delete button below so that the horizontal axis spacing appears identical }}
+  <div class="flex-1 py-1 pl-2">
```

Comment:
The `pl-2` is to resolve the following, where there's more space to the right than left:

<img width="298" alt="Screenshot 2023-06-29 at 1 23 21 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/d35fb2e2-157a-4b3f-be00-0561ff5f0833">

Reply:
I appreciate providing the visuals...makes things so much easier to review.